pyomo/__future__.py

```diff
@@ -0,0 +1,113 @@
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright (c) 2008-2024
+#  National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+
+import pyomo.environ as _environ
+
+__doc__ = """
+Preview capabilities through `pyomo.__future__`
+===============================================
+
+This module provides a uniform interface for gaining access to future
+("preview") capabilities that are either slightly incompatible with the
+current official offering, or are still under development with the
+intent to replace the current offering.
+
+Currently supported `__future__` offerings include:
```

Comment:
```suggestion
Currently supported ``__future__`` offerings include:
```
Is it meant to be italicized or code-blocked?